src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,35 @@
+const ADD_BOOK = 'bookstore/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore/REMOVE_BOOK';
+
+export const addBook = (payload) => ({
+  type: ADD_BOOK,
+  payload,
+});
+
+export const removeBook = (payload) => ({
+  type: REMOVE_BOOK,
+  payload,
+});
+
+const initialState = {
+  books: [],
+};
+
+const books = (state = initialState, { type, payload }) => {
+  switch (type) {
+    case ADD_BOOK:
+      return {
+        ...state,
+        books: [...state.books, payload],
+      };
+    case REMOVE_BOOK:
+      return {
+        ...state,
+        books: [...state.books.filter((book) => book.id !== payload)],
+      };
+    default:
+      return state;
+  }
+};
+
+export default books;
```

Comment:
- [ ] In this activity we want you to simplify the generation of _action_ _types_ and _creators_ by using the `createSlice` from `@reduxjs/toolkit` package. Also, ensure to:
  * Exports by default `booksSlice.reducer`
  * Exports by `addBook` and `removeBook` as actions. 
  
  Example is as below:
```javascript
// ./src/redux/books/booksSlice.js

import { createSlice } from "@reduxjs/toolkit"

export const booksSlice = createSlice({
  name: "books",
  initialState: {
    books: [],
  },
  reducers: {
    addBook: (state, action) => {
      state.books = [
        ...state.books,
        {
          id: Math.floor(Math.random() * 100),
          title: action.payload.title,
          author: action.payload.author,
        },
      ]
    },
    removeBook: (state, action) => {
      state.books = state.books.filter((book) => book.id !== action.id)
    },
  },
})

export const { addBook, removeBook } = booksSlice.actions

export default booksSlice.reducer
```